elements/pfe-progress-indicator/src/pfe-progress-indicator.scss

```diff
@@ -0,0 +1,68 @@
+@import "../../pfe-sass/pfe-sass";
+
+$pfe-progress-indicator--border-light: 4px solid rgba(0, 0, 0, 0.25);
```

Comment:
Instead of hardcoding these values, can you use the variables?  

`$pfe-progress-indicator--border-light: #{pfe-var(surface--border-width--heavy)} #{pfe-var(surface--border-style)} rgba(0, 0, 0, .25);
$pfe-progress-indicator--border-dark: #{pfe-var(surface--border-width--heavy)} #{pfe-var(surface--border-style)} rgba(0, 0, 0, .75);`

Reply:
Def!